frameworks/elastic/build.sh

```diff
@@ -2,16 +2,29 @@
 set -e
 
 FRAMEWORK_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-BUILD_DIR=$FRAMEWORK_DIR/build/distributions
+REPO_ROOT_DIR=$(dirname $(dirname $FRAMEWORK_DIR))
```

Comment:
The `build.sh` for each of the frameworks looks very very similar to me.  Is it worth factoring things out or is the copy/paste due to a desire maintain parity with services outside the mono-repo?

Maybe we could maintain parity and DRY things out a little?

Reply:
There's still some mild variation with e.g. per-service utils, kafka's custom CLI, custom doc URLs, etc... Gave an attempt at deduping but it ended up being more than it was worth